modules/ethereum/src/lib.rs

```diff
@@ -651,7 +702,7 @@ pub(crate) mod tests {
 		best_block: (u64, H256, U256),
 		finalized_block: (u64, H256),
 		oldest_unpruned_block: u64,
-		headers: HashMap<H256, StoredHeader>,
+		headers: HashMap<H256, StoredHeader<u64>>,
```

Comment:
Could we get a typedef `type AccountId = u64`? Now it's a bit confusing to see `u64` for both block numbers and submitters. I know it's only test code, but still.